x-pack/plugins/apm/public/components/shared/EmptyMessage.tsx

```diff
@@ -4,15 +4,20 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React from 'react';
-import PropTypes from 'prop-types';
 import { EuiEmptyPrompt } from '@elastic/eui';
+import React from 'react';
 
-function EmptyMessage({ heading, subheading, hideSubheading }) {
-  if (!subheading) {
-    subheading = 'Try another time range or reset the search filter.';
-  }
+interface Props {
+  heading: string;
+  subheading?: string;
+  hideSubheading?: boolean;
+}
 
+function EmptyMessage({
+  heading = 'No data found.',
+  subheading = 'Try another time range or reset the search filter.',
+  hideSubheading = false
+}: Props) {
```

Comment:
I'd think that the `Prop` interface is redundant when types can be inferred from the default values - but not sure what best practice says.

Reply:
I think I may put this file back to `.js` and remove all of this, and deal with it later.

Reply:
OK so I removed this Props interface and there were no errors, but I only added it because there were errors, so I don't know what happened there. 🤷‍♂️